classes/class-wc-connect-stripe.php

```diff
@@ -218,9 +218,14 @@ public function connection_banner() {
 				return;
 			}
 
+			$options = get_option( self::SETTINGS_OPTION, array() );
+
 			$this->nux->show_nux_banner( array(
-				'title'          => __( 'Connect your account to activate Stripe', 'woocommerce-services' ),
-				'description'    => esc_html( __( 'To start accepting payments with Stripe, you\'ll need to connect your site to a Stripe account.', 'woocommerce-services' ) ),
+				'title'          => __( 'Connect your account', 'woocommerce-services' ),
+				'description'    => wp_kses(
+					sprintf( __( 'It looks like there is an existing Stripe account at <strong>%s</strong>. To start accepting payments with Stripe, you\'ll need to connect it to your store.', 'woocommerce-services' ), $options['email'] ),
```

Comment:
We should check for the `email` key before using it to avoid PHP warnings.

Reply:
I didn't anticipate a case where the banner option would be set but the `email` wouldn't be present, but it's good to add a check. Done in 6b4bed8